client.go

```diff
@@ -365,3 +365,24 @@ func (fc *Client) DoHTTPRequest(ctx context.Context, req *http.Request) (*http.R
 
 	return resp, nil
 }
+
+// GetVersion gets the version information of a homeserver.
+// See https://matrix.org/docs/spec/server_server/r0.1.1.html#get-matrix-federation-v1-version
+func (fc *Client) GetVersion(
```

Comment:
this should be before the things it calls (ie, `DoRequestAndParseResponse`), and anyway conceptually it sits alongside `GetServerKeys` etc